src/Service/SimXMLToZGWService.php

```diff
@@ -3,16 +3,20 @@
 namespace CommonGateway\SimXMLToZGWBundle\Service;
 
 use Adbar\Dot;
+use App\Entity\Endpoint;
 use App\Entity\Entity;
+use App\Entity\File;
 use App\Entity\Mapping;
 use App\Entity\ObjectEntity;
+use App\Entity\Value;
```

Comment:
Volgensmij heb je deze use niet nodig

Reply:
Ga ik testen, kan zijn dat deze weer vervallen is inderdaad